src/components/loginform/LoginForm.js

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import CustomizableButton from "../common/CustomizableButton";
+import styles from "./loginform.module.css" 
+
+function LoginForm() {
+
+    const [formData, setFormData] = useState({
+        email: '',
+        password: '',
+      });
+      const [errors, setErrors] = useState({});
+    
+      const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData({
+          ...formData,
+          [name]: value,
+        });
+      };
+    
+      const validateForm = () => {
```

Comment:
This function can be simplified. I would split this validation logic into separate functions. 1 to validate email, one for password and one to call both and determine if the form is valid.

Taylor did something similar in her most resent PR and I asked her to abstract the validation logic into a separate module and import the functions. Once she does that I think you could just import the validation functions and use them here. Please connect with her on this.

As a rule of thumb, try to keep functions as simple as possible. They should have just one functionality. They should take some arguments perform one action and return something as a result. Keeping them simple makes them easier to read, modify and test